constraints.md

```diff
@@ -6,67 +6,7 @@ aliases: ['/docs-cn/dev/reference/sql/constraints/']
 
 # 约束
 
-TiDB 支持的基本约束与 MySQL 的基本相同，但有以下区别：
-
-- 默认对唯一约束进行[惰性检查](/transaction-overview.md#事务的惰性检查)。通过在事务提交时再进行批量检查，TiDB 能够减少网络开销、提升性能。您可通过设置 `tidb_constraint_check_in_place` 为 `TRUE` 改变此行为。
-
-- TiDB 支持创建外键约束，但不会在 DML 语句中对外键进行约束（即外键约束不生效）。
-
-## 外键约束
-
-目前，TiDB 支持创建外键。例如：
-
-```sql
-CREATE TABLE users (
- id INT NOT NULL PRIMARY KEY AUTO_INCREMENT,
- doc JSON
-);
-CREATE TABLE orders (
- id INT NOT NULL PRIMARY KEY AUTO_INCREMENT,
- user_id INT NOT NULL,
- doc JSON,
- FOREIGN KEY fk_user_id (user_id) REFERENCES users(id)
-);
-```
-
-{{< copyable "sql" >}}
-
-```sql
-SELECT table_name, column_name, constraint_name, referenced_table_name, referenced_column_name
-FROM information_schema.key_column_usage WHERE table_name IN ('users', 'orders');
-```
-
-```
-+------------+-------------+-----------------+-----------------------+------------------------+
-| table_name | column_name | constraint_name | referenced_table_name | referenced_column_name |
-+------------+-------------+-----------------+-----------------------+------------------------+
-| users      | id          | PRIMARY         | NULL                  | NULL                   |
-| orders     | id          | PRIMARY         | NULL                  | NULL                   |
-| orders     | user_id     | fk_user_id      | users                 | id                     |
-+------------+-------------+-----------------+-----------------------+------------------------+
-3 rows in set (0.00 sec)
-```
-
-TiDB 也支持使用 `ALTER TABLE` 命令来删除外键（`DROP FOREIGN KEY`）和添加外键（`ADD FOREIGN KEY`）：
-
-{{< copyable "sql" >}}
-
-```sql
-ALTER TABLE orders DROP FOREIGN KEY fk_user_id;
-ALTER TABLE orders ADD FOREIGN KEY fk_user_id (user_id) REFERENCES users(id);
-```
-
-### 注意
-
-* TiDB 支持外键是为了在将其他数据库迁移到 TiDB 时，不会因为此语法报错。但是，TiDB 不会在 DML 语句中对外键进行约束。例如，即使 `users` 表中不存在 `id=123` 的记录，下列事务也能提交成功：
-
-    ```
-    START TRANSACTION;
-    INSERT INTO orders (user_id, doc) VALUES (123, NULL);
-    COMMIT;
-    ```
-
-* TiDB 在执行 `SHOW CREATE TABLE` 语句的结果中不显示外键信息。
+TiDB 支持的约束与 MySQL 的基本相同。
```

Comment:
```suggestion
TiDB 支持的约束与 MySQL 的基本相同。不同约束的详细描述如下：
```

Reply:
We don't have the part for the different constraints.